File: typescipt/src/compiler/JavaScript/builtInFunctions/minuteFunctionCall.ts
import {SingleArgumentFunctionCall} from "./singleArgumentFunctionCall";
import {MinuteFunction} from "../../../language/expressions/functions/minuteFunction";

export class MinuteFunctionCall extends SingleArgumentFunctionCall {
   protected override className = "BuiltInDateFunctions";
   protected override methodName = "minute";

   constructor(functionNode: MinuteFunction) {
      super(functionNode);
   }
}

File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/MinuteFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions;

internal class MinuteFunctionCall : SingleArgumentFunctionCall<MinuteFunction>
{
    protected override string ClassName => nameof(BuiltInDateFunctions);
    protected override string MethodName => nameof(BuiltInDateFunctions.Minute);
}